df-valve/src/main/java/com/hashmapinc/tempus/witsml/valve/dot/DotTranslator.java

```diff
@@ -15,9 +15,15 @@
  */
```

Comment:
Added method to take the query json object parsed from the soap client and merge it with the `GET` result from `DoT`. 

This sweet jamba juice relies on upgraded JSON support to `WitsmlObjects` library. This was built by yours truly in version `1.1.4`.